src/main/kotlin/study/step3/race/reader/ActualInputReader.kt

```diff
@@ -0,0 +1,5 @@
+package study.step3.race.reader
+
+class ActualInputReader : InputReader {
+    override fun raedLine() = readlnOrNull()
```

Comment:
```suggestion
    override fun raedLine() = readln()
```

readln()을 사용하여 null을 방지하는 건 어떨까요?

참고 
- [kotlin readln](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/readln.html)
- [kotlin 1.6.0 released](https://blog.jetbrains.com/ko/kotlin/2021/12/kotlin-1-6-0-is-released/)

Reply:
확실히 굳이 Null 을 허가해줄 필요가 없는데, 허가해준 코드였네요. 수정하겠습니다.